crates/teloxide/src/dispatching/dialogue/get_chat_id.rs

```diff
@@ -23,3 +26,68 @@ impl GetChatId for Update {
         self.chat().map(|chat| chat.id)
     }
 }
+
+impl GetChatId for Recipient {
```

Comment:
This is meaningless — `Recipient` is only ever passed to telegram, not the other way around